alert-common/src/main/java/com/synopsys/integration/alert/common/descriptor/config/ui/UIConfig.java

```diff
@@ -48,6 +48,8 @@ public UIConfig(String label, String description, String urlName) {
 
     public abstract List<ConfigField> createFields();
 
+    public abstract boolean hasFields();
```

Comment:
Since this is just a boolean, should we have it passed through the constructor and remove abstract from this method?